### PR TITLE
New Plugin: Azure Maps basemap provider

### PR DESCRIPTION
**Pull request by rbrundritt**
Adding the Azure Maps Leaflet plugin to the list of basemap provider plugins. The plugin project can be found here: https://github.com/Azure-Samples/azure-maps-leaflet

### CODE REVIEW

**Review by johnd0e — APPROVED** [review]
Cool.
Note that your sample code uses outdated 1.3.1 version of Leaflet.

File: docs/plugins.md [hunk 1]
```diff
@@ -234,6 +234,16 @@ Ready-to-go basemaps, with little or no configuration at all.
 			<a href="https://github.com/rkaravia">Roman Karavia</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/Azure-Samples/azure-maps-leaflet">Azure Maps Leaflet plugin</a>
+		</td><td>
+			A leafletjs plugin that makes it easy to overlay all the different tile layers available from the [Azure Maps](https://azure.com/maps). Supports using an Azure Maps subscription key or Azure Active Directory for authentication.
```
Comment (mourner) [inline docs/plugins.md:241]: I think the Markdown link should be converted to HTML because it's inside an HTML table — it won't be processed.

Comment (rbrundritt) [reply docs/plugins.md]: Good catch. I'm so used to using markdown in my descriptions. I've corrected this.

**Comment by rbrundritt** [comment]
> Cool.
> Note that your sample code uses outdated 1.3.1 version of Leaflet.

Right. I had been testing with multiple leaflet versions. I'll update the samples to use the latest.

**Comment by johnd0e** [comment]
@rbrundritt 
Please resolve conflicts

File: docs/plugins.md [hunk 2]
```diff
@@ -234,6 +234,16 @@ Ready-to-go basemaps, with little or no configuration at all.
 			<a href="https://github.com/rkaravia">Roman Karavia</a>
 		</td>
 	</tr>
+  <tr>
```
Comment (johnd0e) [inline docs/plugins.md:237]: Please use tabs for indent

**Review by johnd0e — CHANGES_REQUESTED** [review]

**Review by Falke-Design — APPROVED** [review]

File: docs/plugins.md (hunk 2 above)
Comment (johnd0e) [reply docs/plugins.md:237]: Please use tabs for indent

**Review by johnd0e — CHANGES_REQUESTED** [review]

File: docs/plugins.md (hunk 2 above)
Comment (rbrundritt) [reply docs/plugins.md]: Looks like this was just for one single line. This has now been updated.

**Review by johnd0e — APPROVED** [review]

**Pull request merged**